packages/@uppy/companion/src/server/helpers/utils.js

```diff
@@ -43,7 +43,7 @@ module.exports.getURLBuilder = (options) => {
    *
    * @param {string} path the tail path of the url
    * @param {boolean} isExternal if the url is for the external world
-   * @param {boolean=} excludeHost if the server domain and protocol should be included
```

Comment:
Out of curiosity, what does this fix exactly? isn't `{boolean=}` a valid JSDoc format?

Reply:
Not really, it's a Google Closure Compiler specific syntax, see https://jsdoc.app/tags-param.html#optional-parameters-and-default-values
Our linter is in TypeScript mode for JSDoc, and reports it as an error if we try to enforce that rule.
https://github.com/transloadit/uppy/blob/456c2b7422186effe23babdcded0be60c064f5f7/.eslintrc.js#L137-L139

Reply:
@mifi 
```console
$ corepack yarn eslint packages/@uppy/companion/src/server/helpers/utils.js

…/uppy/packages/@uppy/companion/src/server/helpers/utils.js
  30:5  warning  Arrow function expected no return value             consistent-return
  46:0  warning  Syntax should not be Google Closure Compiler style  jsdoc/check-syntax
```

Reply:
Alright thanks for clarifying!